app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -116,6 +118,31 @@ const MeetingEditor = () => {
     })
   );
 
+  const updateReportValue = useCallback(
+    (newReportValue) => {
+      const debouncedFunction = debounce(() => {
+        const storedReport = localStorage.getItem(`meeting-${meetingId}-temp`);
+        if (!storedReport || storedReport.length < newReportValue.length) {
+          setReportValue(newReportValue);
+        }
+      }, 4500);
+
+      debouncedFunction();
+    },
+    [meetingId]
+  );
+
+  useEffect(() => {
+    if (meeting && meetingId) {
+      const storedReport = localStorage.getItem(
+        `meeting-${meeting?.id}-report`
+      );
+      if (!storedReport || storedReport.length < reportValue.length) {
+        localStorage.setItem(`meeting-${meeting?.id}-report`, reportValue);
+      }
+    }
+  }, [reportValue, meeting, meetingId]);
```

Comment:
It would be nice with a tiny comment explaining the code here and why it is needed